### ISSUE TITLE
the trap contains... translation issue.

### ISSUE THREAD

**Issue by skj3gg** [doublespeakgames/adarkroom#335]
@BTLmovies pointed out that "stoffundvereinzeltezähne" is wrong and should be "stoff und vereinzelte zähne" in the notifications.

My basic German knowledge is telling me that the spaces in the `"this trap contains "` code isn't keeping the spaces.

![](https://cloud.githubusercontent.com/assets/12112956/7333432/f69a3e4e-eb6f-11e4-9d26-e8bb20e7a8a5.png)

**Comment by BTLmovies** [comment]
Thank you :) :+1:

**Comment by JustinLee99** [comment]
I think I solved this.  Just as easy as adding spaces into the German translation to match the format presented in the Check Trap function.  I tested it out by checking the traps a number of times with different amounts of traps set up to make sure it works for all possible yields.  

![fix1](https://cloud.githubusercontent.com/assets/11264070/7339561/c6a708e4-ec27-11e4-95b0-ad4826c343fe.png)

I haven't encountered any problems with words being merged into each other at all after this fix.
